src/action/common/configure_init_service.rs

```diff
@@ -157,23 +161,38 @@ impl Action for ConfigureInitService {
                 )
                 .await?;
 
-                execute_command(
-                    Command::new("systemctl")
-                        .process_group(0)
-                        .arg("link")
-                        .arg(SERVICE_SRC)
-                        .stdin(std::process::Stdio::null()),
-                )
-                .await?;
+                if Path::new(SERVICE_DEST).exists() {
+                    // TODO: ask user if we can remove it...? do we have a way
```

Comment:
I don't think we are currently structured to be able to have some interactivity within the plans... We do have a `prompt` function but that's part of the CLI and it would feel weird putting that in here...

Reply:
We have discussed adding some UX for curing to prompt users, but as of right now -- no support.